plugin/src/main/java/com/denizenscript/denizen/events/BukkitScriptEvent.java

```diff
@@ -163,6 +163,22 @@ public static boolean couldMatchEnum(String text, final Enum<?>[] enumVals) {
         return false;
     }
 
+    public static boolean couldMatchRegistry(String text, Registry<?> registry) {
+        String textLower = CoreUtilities.toLowerCase(text);
+        MatchHelper matcher = isAdvancedMatchable(text) ? createMatcher(text) : null;
```

Comment:
this alternative handling is redundant the way this is implemented. Could swap the non-matchable version to be a `registry.get` call, or just drop the ternary and always use matcher